### DISCUSSION TITLE
Realtime not connecting

### DISCUSSION THREAD

**Discussion by sebastienaugsburger** [supabase/supabase#21032] [Questions]
This is about Supabase Swift.

After updating to latest package versions, my realtime is no longer connecting. The warning printed includes Code -1011"There was a bad response from server." and the next line includes NSERRORFailingURLStringKey. Is my issue because of the update(v1 to v2) and how can I fix this? I already have realtime enabled and removed RLS for the table.

**Comment by GaryAustin1** [comment]
https://github.com/supabase-community/supabase-swift/issues/122
Just FYI on that error.  That link is not related to realtime but has the same error.
I know nothing about the Swift code, but did not find a realtime link to that code in searching.
